src/components/UserInfo.js

```diff
@@ -14,4 +15,8 @@ export default class UserInfo {
     this.name.textContent = name;
     this.description.textContent = description;
   }
+
+  setUserImage({ avatar }) {
```

Comment:
It's great to have a setter for the avatar as it's controlled by a different form